components/catalogs/front/src/components/ProductSelection/components/AnyLocaleInput.tsx

```diff
@@ -26,7 +26,7 @@ const AnyLocaleInput: FC<Props> = ({state, onChange, isInvalid}) => {
             onChange={v => onChange({...state, locale: v})}
             clearable={false}
             invalid={isInvalid}
-            placeholder={translate('akeneo_catalogs.product_selection.locale.label')}
+            placeholder={translate('akeneo_catalogs.product_selection.filters.locale.label')}
```

Comment:
```suggestion
            placeholder={translate('akeneo_catalogs.product_selection.locale.label')}
```

this is not filters, this is input for product selection